iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -616,11 +616,67 @@ static LogicalResult setRootConfig(
       entryPointFn, fftOp, tileSizes, DispatchLoweringPassPipeline::CPUDefault);
 }
 
-/// Sets the lowering configuration for a generic op to use
+static void setX86WorkgroupTileSizes(
+    linalg::GenericOp genericOp, unsigned numLoops,
+    ArrayRef<int64_t> flowTileSizes, ArrayRef<int64_t> minTileSizes,
+    ArrayRef<int64_t> maxTileSizes,
+    SmallVectorImpl<int64_t> &parallelTileSizes) {
```

Comment:
We should rename `parallelTileSizes` to something else. Maybe name it to `res` and return the `SmallVector<int64_t>`?